reference-tests/src/test/java/net/consensys/linea/BlockchainReferenceTestTools.java

```diff
@@ -77,14 +77,22 @@ public class BlockchainReferenceTestTools {
     if (NETWORKS_TO_RUN.isEmpty()) {
       PARAMS.ignoreAll();
     }
+    // ignore tests that are failing in Besu too
+    PARAMS.ignore("RevertInCreateInInitCreate2_d0g0v0_London[London]");//failing in Besu too
+    PARAMS.ignore("RevertInCreateInInit_d0g0v0_London[London]");//failing in Besu too
+    PARAMS.ignore("create2collisionStorage_d0g0v0_London[London]");//failing in Besu too
+    PARAMS.ignore("create2collisionStorage_d1g0v0_London[London]");//failing in Besu too
+    PARAMS.ignore("create2collisionStorage_d2g0v0_London[London]");//failing in Besu too
+    PARAMS.ignore("dynamicAccountOverwriteEmpty_d0g0v0_London[London]");//failing in Besu too
+
     // ignore tests that are failing because there is an account with nonce 0 and
     // non empty code which can't happen in Linea since we are post LONDON only.
-    PARAMS.ignore("InitCollision_d0g0v0_London[London]");
-    PARAMS.ignore("InitCollision_d1g0v0_London[London]");
-    PARAMS.ignore("InitCollision_d2g0v0_London[London]");
-    PARAMS.ignore("InitCollision_d3g0v0_London[London]");
-    PARAMS.ignore("RevertInCreateInInitCreate2_d0g0v0_London[London]");
-    PARAMS.ignore("RevertInCreateInInit_d0g0v0_London[London]");
+    PARAMS.ignore("InitCollision_d0g0v0_London[London]"); // non empty + nonce 0
```

Comment:
there is already he comment about non empty + nonce = 0 at the begining of the section. More readable